openid-4-verifiable-presentations-1_0.md

```diff
@@ -559,11 +565,276 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+# Verifiable Presentation Query Language {#vp_query}
+
+The Verifiable Presentation Query Language (VP Query) is a JSON-encoded query
+language that allows the Verifier to request the Wallet to present Verifiable
+Credentials that match the query. The Verifier MAY encode constraints on the
+combinations of credentials and claims that are requested. The Wallet evaluates the
+query against the Verifiable Credentials it holds and returns the Verifiable
+Presentations that match the query.
+
+A valid VP Query is defined as a JSON-encoded object with the following
+top-level properties:
+
+`expect_credentials`:
```

Comment:
A general nit across the query syntax proposed, I would suggest we just drop `expect_` from all parts of the syntax, it is after all, inside a query parameter that describes the surrounding data structures purpose.

Reply:
I dropped the `expect_` prefix throughout the PR.